src/sklearn_evaluation/plot/classification.py

```diff
@@ -51,12 +50,25 @@ def __init__(self, cm, target_names) -> None:
 
 class ConfusionMatrixAdd(Plot):
     def __init__(self, a, b, target_names) -> None:
-        self.figure = Figure()
+        self.figure = plt.figure()
```

Comment:
what's the difference between using `Figure` and `figure`? Will this change anything?